packages/patternfly-4/react-core/src/components/Label/Label.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Label/styles.css';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const Label = ({ className, children, isCompact }) => (
+  <span className={css(styles.label, className, isCompact && getModifier(styles, isCompact && 'compact'))}>
```

Comment:
make this `isCompact && styles.modifiers.compact` It will typecheck. getModifier doesn't type check at all, and it doesn't really help out much here. It was only meant for times were we want the user to pass in a variant and default it if they pass the wrong thing in.